www/examples/example13-opacity-transition.js

```diff
@@ -1,13 +1,14 @@
 var div = document.querySelectorAll("div")[0];
 
 function start_fade_out(e) {
-	div.style = "opacity:0.1";
-  e.preventDefault();
+    div.style = "opacity:0.1";
+    e.preventDefault();
 }
 function start_fade_in(e) {
-	div.style = "opacity:0.999";
-  e.preventDefault();
+    div.style = "opacity:0.999";
+    e.preventDefault();
```

Comment:
Otherwise the indentation changes kind of randomly, and this is transcluded in the book.